api/nodes/views.py

```diff
@@ -360,7 +364,10 @@ class NodeDetail(generics.RetrieveUpdateDestroyAPIView, NodeMixin):
 
     # overrides RetrieveUpdateDestroyAPIView
     def get_object(self):
-        return self.get_node()
+        node = self.get_node()
+        if node.is_registration:
```

Comment:
Discussed in person with Brian: let's allow registrations to be listed via the `/nodes/` endpoints for now. AFAIK, it is ok to have the same stored object have multiple representations. So if a registration is requested through a `/nodes/` URL, it's `self` link will be the `/nodes/<id>`. If requesting through `/registrations/` -> `/registrations/<id>`, etc.